packages/sheet/src/sheet.test.ts

```diff
@@ -80,4 +81,37 @@ describe("Sheet class", () => {
     // Assert
     expect(cssString).toEqual(expectedCSS);
   });
+
+  describe("Sheet placeholders", () => {
+    beforeAll(() => {
+      theme.setUserTheme({
+        breakpoints: {
+          sm: "1000px",
+        },
+        colors: {
+          primary: "red",
+          secondary: "blue",
+        },
+        zIndices: {
+          modal: "1000",
+        },
+      });
```

Comment:
There seems to be a mistake in this test because the return value of [createTheme](https://github.com/kuma-ui/kuma-ui/blob/e32721f5bdaa5e90b1a4f781483b25ededcc4c34/packages/core/src/theme.ts#L83-L101) API is passed as an argument to setUserTheme.

```suggestion
      theme.setUserTheme({
        breakpoints: {
          "breakpoints.sm": "1000px",
        },
        colors: {
          "colors.primary": "red",
          "colors.secondary": "blue",
        },
        zIndices: {
          "zIndices.modal": "1000",
        },
      });
```

Reply:
The idea was to prevent the conflicts and to keep the existing structure of tokens.

So the placehorlders are build like <tokenType>.<tokenName>

The approach you are proposing will be less intuitive and more error prone as you can easily give different tokens the same name

Reply:
Hmmm, what does that mean ?
Themes defined in kuma.config.ts are exported as return values of createTheme API. The return value is passed as an argument to theme.setUserTheme. the return value of the createTheme API is converted to the structure I have suggested. Have you checked the following codes?
https://github.com/kuma-ui/kuma-ui/blob/e32721f5bdaa5e90b1a4f781483b25ededcc4c34/packages/core/src/theme.ts#L83-L101

Reply:
If kuma.config.ts was configured as follows,

```ts
// kuma.config.ts
import { createTheme } from "@kuma-ui/core";

const theme = createTheme({
  colors: {
    red: "red",
});
// theme: {
//   colors: {
//     "colors.red": "red",
//   },
// }

type UserTheme = typeof theme;

declare module "@kuma-ui/core" {
  export interface Theme extends UserTheme {}
}

export default theme;
```

Your code does not work with reference to theme as follows.

```ts
<Box className={css`color: t("colors.red")`}>test</Box>
```

Your code works by referencing theme as follows.

```ts
<Box className={css`color: t("colors.colors.red")`}>test</Box>
```

Reply:
Please see createTheme test: 
https://github.com/kuma-ui/kuma-ui/blob/e32721f5bdaa5e90b1a4f781483b25ededcc4c34/packages/core/src/theme.test.ts